apps/et/et-ccd-case-migration/et-ccd-case-migration.yaml

```diff
@@ -8,9 +8,9 @@ spec:
     job:
       suspend: true
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/et/migration:prod-e00ecc6-20240423073055 #{"$imagepolicy": "flux-system:et-ccd-case-migration"}
+      image: hmctspublic.azurecr.io/et/migration:pr-201-01ab268-20240424170425 #{"$imagepolicy": "flux-system:et-ccd-case-migration"}
```

Comment:
this should always be the prod version

Reply:
and also, this line either needs to be in demo.yaml or in the demo-00 and demo-01 files

Reply:
reverted